.teamcity/_Self/templates/TestTemplateBase.kt

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+package _self.templates
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.freeDiskSpace
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+
+open class TestTemplateBase : Template({
+    name = "Python Test Template Base Class"
+    id = RelativeId("TestTemplateBase")
+
+    artifactRules = "+:%project.directory%/htmlcov => coverage.zip"
+    option("shouldFailBuildOnAnyErrorMessage", "true")
+
+    vcs {
+        root(DslContext.settingsRoot, "%vcs.checkout.rules%")
+    }
+
+    steps {
+        script {
+            name = "Install"
+            formatStderrAsError = true
+            workingDir = "eng"
+            executionMode = BuildStep.ExecutionMode.RUN_ON_SUCCESS
+            scriptContent = """
+                    python ./build.py install ../%project.directory%
+                """.trimIndent()
```

Comment:
Here's why I created a Python script to help automate the build process: this is not Windows specific. This TeamCity build configuration should run on Linux as well. Not that we're trying to run it there right now, but it is nice to know that we have that flexibility.